doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -76,7 +77,8 @@ Initialization
 
 Ideally, once the device has connected to the cloud, the application must call the P-GPS initialization function.
 If a connection is not available, initialization must still be called.
-In this case, predictions will be unavailable if all valid predictions have expired, until a connection is established to the cloud in the future.
+If :kconfig:`CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT` is enabled, the initialization function will not automatically download missing P-GPS data.
+In these cases, predictions might be unavailable until a connection is established to the cloud in the future.
```

Comment:
```suggestion
If the :kconfig:`CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT` option is enabled, the initialization function does not automatically download missing P-GPS data.
In these cases, predictions might be unavailable until a connection is established to the cloud.
```

Reply:
That's fine, though some of that language was there before...